features/networking/service.feature

```diff
@@ -479,3 +479,56 @@ Feature: Service related networking scenarios
     Then the step should succeed
     And the output should not contain "mtu-too-small"
     """
+
+  # @author zzhao@redhat.com
+  # @case_id OCP-33848
+  @admin
+  @destructive
+  Scenario: User can expand the nodePort range by patch the serviceNodePortRange in network
+    Given I have a project
+    And evaluation of `rand(32676..33000)` is stored in the :port clipboard
+    Given as admin I successfully merge patch resource "networks.config.openshift.io/cluster" with:
+      | {"spec":{"serviceNodePortRange": "30000-33000"}} |
```

Comment:
We need clean-up for this change to restore the cluster.

Reply:
@liangxia  User cannot decrease the nodePort range in post action, So once the serviceNodePortRange is expand, it cannot be restored to origin(30000-32676).

Reply:
Ack